### DISCUSSION TITLE
HMS-1600 4T keine Verbindung

### DISCUSSION THREAD

**Discussion by ulrich-lacher** [hoylabs/OpenDTU-OnBattery#1022] [Q&A]
Hallo zusammen. 
Bisheriges System: openDTU Onbattery mit DPL - HM-600 - 48V Batterie(Eigenbau)
Funktioniert seit einem Jahr problemlos. Danke für eure super Arbeit.
Am WE wollte ich den HM-600 gegen HMS-1600 4T tauschen. Zuerst ließ sich die SN nicht eintragen, hier gesucht und Lösung gefunden mit neuer Firmware.
Allerdings bekomme ich keine Daten vom Wechselrichter. WR ist angeschlossen und blinkt grün. OpenDTU zeigt Einspeisung an.
Der bisherige WR ist abgeklemmt.
Im live Bild (Browser) kann ich auswählen WR HM-600 und HMS1600.
HMS 1600 ausgewählt, WR Informationen werden nicht empfangen.
Was übersehe ich? Muss ich den  HM-600 löschen?
Danke schon mal für die Antworten.

**Comment by spcqike** [comment]
Der HMS braucht ein anderes Funk Modul. Nicht den NRf24.

**Reply by ulrich-lacher** [reply]
Super Hinweis, Danke👍 Weißt du auch welches?
Kann ich es einfach austauschen oder brauche ich dann andere Firmware?

**Reply by spcqike** [reply]
Du brauchst ein CMT2300A. 

Es gab mal ne tabellarische Übersicht, in der stand, welcher wechselrichter welches funkmoduk benötigt. Aber die finde ich grade nicht mehr 

im Wiki steht noch etwas dazu
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Device-Profiles

Nein eine neue Firmware brauchst du nicht. Du brauchst nur das passende pin-Mapping. 

Der NRF und der CMT sollten pin kompatibel sein. Guck aber bei deinem Modul vorsichtshalber nochmal nach.

**Reply by schlimmchen** [reply]
> Es gab mal ne tabellarische Übersicht, in der stand, welcher wechselrichter welches funkmoduk benötigt. Aber die finde ich grade nicht mehr

Hier: https://www.opendtu.solar/hardware/inverter_overview/

**Reply by ulrich-lacher** [reply]
Danke für eure Infos
